{
 "PRed":[
  {
   "PR":{
    "__lazy_json__":"pr_json/484411792.json"
   },
   "data":{
    "bot_rerun":false,
    "migrator_name":"Version",
    "migrator_version":0,
    "version":"5.5.1"
   },
   "keys":[
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR":{
    "__lazy_json__":"pr_json/485211147.json"
   },
   "data":{
    "bot_rerun":false,
    "migrator_name":"Version",
    "migrator_version":0,
    "version":"5.5.2"
   },
   "keys":[
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR":{
    "__lazy_json__":"pr_json/485960692.json"
   },
   "data":{
    "bot_rerun":false,
    "migrator_name":"Version",
    "migrator_version":0,
    "version":"5.6.0"
   },
   "keys":[
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR":{
    "__lazy_json__":"pr_json/488671398.json"
   },
   "data":{
    "bot_rerun":false,
    "migrator_name":"Version",
    "migrator_version":0,
    "version":"5.6.1"
   },
   "keys":[
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  },
  {
   "PR":{
    "__lazy_json__":"pr_json/489960806.json"
   },
   "data":{
    "bot_rerun":false,
    "migrator_name":"Version",
    "migrator_version":0,
    "version":"5.6.2"
   },
   "keys":[
    "bot_rerun",
    "migrator_name",
    "migrator_version",
    "version"
   ]
  }
 ],
 "archived":false,
 "bad":false,
 "conda-forge.yml":{},
 "feedstock_name":"django-axes",
 "hash_type":"sha256",
 "linux_64_meta_yaml":{
  "about":{
   "home":"https://pypi.org/project/django-axes/",
   "license":"MIT",
   "license_file":"LICENSE",
   "summary":"django-axes, User lockout in django apps."
  },
  "build":{
   "noarch":"python",
   "number":"0",
   "script":"-m pip install . -vv"
  },
  "extra":{
   "recipe-maintainers":[
    "swainn",
    "gagelarsen"
   ]
  },
  "package":{
   "name":"django-axes",
   "version":"5.6.1"
  },
  "requirements":{
   "host":[
    "pip",
    "python",
    "setuptools",
    "setuptools_scm"
   ],
   "run":[
    "setuptools",
    "python",
    "django >=2.0",
    "django-appconf >=1.0.3",
    "django-ipware >=3,<4"
   ]
  },
  "source":{
   "sha256":"3d80b572f8eec549a2df453f53e6b4ee35faeffb07723d3a11332b9626d02d94",
   "url":"https://pypi.io/packages/source/d/django-axes/django-axes-5.6.1.tar.gz"
  },
  "test":{
   "imports":[
    "axes"
   ]
  }
 },
 "linux_64_requirements":{
  "build":{
   "__set__":true,
   "elements":[]
  },
  "host":{
   "__set__":true,
   "elements":[
    "pip",
    "python",
    "setuptools",
    "setuptools_scm"
   ]
  },
  "run":{
   "__set__":true,
   "elements":[
    "django",
    "django-appconf",
    "django-ipware",
    "python",
    "setuptools"
   ]
  },
  "test":{
   "__set__":true,
   "elements":[]
  }
 },
 "meta_yaml":{
  "about":{
   "home":"https://pypi.org/project/django-axes/",
   "license":"MIT",
   "license_file":"LICENSE",
   "summary":"django-axes, User lockout in django apps."
  },
  "build":{
   "noarch":"python",
   "number":"0",
   "script":"-m pip install . -vv"
  },
  "extra":{
   "recipe-maintainers":[
    "swainn",
    "gagelarsen"
   ]
  },
  "package":{
   "name":"django-axes",
   "version":"5.6.1"
  },
  "requirements":{
   "host":[
    "pip",
    "python",
    "setuptools",
    "setuptools_scm"
   ],
   "run":[
    "setuptools",
    "python",
    "django >=2.0",
    "django-appconf >=1.0.3",
    "django-ipware >=3,<4"
   ]
  },
  "source":{
   "sha256":"3d80b572f8eec549a2df453f53e6b4ee35faeffb07723d3a11332b9626d02d94",
   "url":"https://pypi.io/packages/source/d/django-axes/django-axes-5.6.1.tar.gz"
  },
  "test":{
   "imports":[
    "axes"
   ]
  }
 },
 "name":"django-axes",
 "new_version":"5.6.2",
 "new_version_attempts":{
  "5.5.1":1,
  "5.5.2":1,
  "5.6.0":1,
  "5.6.1":1,
  "5.6.2":1
 },
 "new_version_errors":{},
 "pinning_version":"2020.09.19.20.20.14",
 "raw_meta_yaml":"{% set name = \"django-axes\" %}\n{% set version = \"5.6.1\" %}\n\npackage:\n  name: {{ name|lower }}\n  version: {{ version }}\n\nsource:\n  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz\n  sha256: 3d80b572f8eec549a2df453f53e6b4ee35faeffb07723d3a11332b9626d02d94\n\nbuild:\n  number: 0\n  noarch: python\n  script: {{ PYTHON }} -m pip install . -vv\n\nrequirements:\n  host:\n    - pip\n    - python\n    - setuptools\n    - setuptools_scm\n  run:\n    - setuptools\n    - python\n    - django >=2.0\n    - django-appconf >=1.0.3\n    - django-ipware >=3,<4\n\ntest:\n  imports:\n    - axes\n\nabout:\n  home: https://pypi.org/project/django-axes/\n  summary: django-axes, User lockout in django apps.\n  license: MIT\n  license_file: LICENSE\n\nextra:\n  recipe-maintainers:\n    - swainn\n    - gagelarsen\n",
 "req":{
  "__set__":true,
  "elements":[
   "django",
   "django-appconf",
   "django-ipware",
   "pip",
   "python",
   "setuptools",
   "setuptools_scm"
  ]
 },
 "requirements":{
  "build":{
   "__set__":true,
   "elements":[]
  },
  "host":{
   "__set__":true,
   "elements":[
    "pip",
    "python",
    "setuptools",
    "setuptools_scm"
   ]
  },
  "run":{
   "__set__":true,
   "elements":[
    "django",
    "django-appconf",
    "django-ipware",
    "python",
    "setuptools"
   ]
  },
  "test":{
   "__set__":true,
   "elements":[]
  }
 },
 "smithy_version":"No azure token. Create a token and\nput it in ~/.conda-smithy/azure.token\n3.7.10",
 "strong_exports":false,
 "total_requirements":{
  "build":{
   "__set__":true,
   "elements":[]
  },
  "host":{
   "__set__":true,
   "elements":[
    "pip",
    "python",
    "setuptools",
    "setuptools_scm"
   ]
  },
  "run":{
   "__set__":true,
   "elements":[
    "django >=2.0",
    "django-appconf >=1.0.3",
    "django-ipware >=3,<4",
    "python",
    "setuptools"
   ]
  },
  "test":{
   "__set__":true,
   "elements":[]
  }
 },
 "url":"https://pypi.io/packages/source/d/django-axes/django-axes-5.5.0.tar.gz",
 "version":"5.6.1"
}